### DISCUSSION TITLE
New Team Member: Welcome

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#462] [Announcements]
@Akaflieg-Freiburg/enroute 

Dear all,

I am happy to announce that yet another colleague has joined our team. 

Christian Engelhardt is a PPL pilot in southern Germany. He studied electrical engineering and works as an Embedded SW Engineer.

@charlylima Welcome!

Christian started the work on density altitude calculation, which will likely appear in release 2.31.17 soon. He is currently working on creating a ["development docker"](https://github.com/charlylima/enroute/tree/dockerfile) that has Qt and the dependencies pre-configured for easy compilation. You can even use the docker container to launch the Linux development environment on MS Windows with WSL2. For the future, think about helping with topics such as GPS logging and automatic logging of start and landing time.

Best wishes from Foggy Freiburg,

Stefan.

**Comment by TTC70** [comment]
@charlylima welcome to the EFN family 🎉🎊🎉🎊🎉🎊🎉🎊🎉
